### DISCUSSION TITLE
Altura CSS

### DISCUSSION THREAD

**Discussion by Canti21** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#538] [Q&A]
Hola! Tengo un problema con una imagen.

Tengo una seccion divida en 2 columnas con grid, y en una columna tengo una imagen con una altura de 955px, y quiero que su altura sea el 70% de la altura de la otra columna.

El problema es que aunque reduzca el tamaño de la imagen, el div conserva la altura de la imagen original y me pone un espacio en blanco muy feo.

Hay forma solucionar eso?

![null](https://user-images.githubusercontent.com/81689973/160257108-78e4e100-3982-4709-a2be-603571ad7752.png)

**Comment by MarcosSilvaG** [comment]
podrias mostrar el código de cómo estás haciendo eso?

**Reply by Canti21** [reply]
HTML:
```
<section class="product">
    <div class="image-column">
        <figure>
            <img src="./img/image.png" alt="">
        </figure>
    </div>
    <div class="product-info">
        <h2>Pastel Triple Chocolate</h2>
        <p>Lorem Ipsum</p>
        <p class="price">$220</p>
    </div>
</section>
```

CSS:
```
.product
{
    display: grid;
    grid-template-columns: 1fr 1fr;
}

.image-column
{
    background: var(--background-color);
}

.image-column figure
{
    height: 100%;
    display: flex;
    justify-content: center;
    align-items: flex-end;
}

.image-column img
{
    height: 70%;
}

.product-info
{
    padding: 0 70px;
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-content: flex-end;
    margin-right: 40px;
}
```

**Reply by carlos-gmx3** [reply]
creo q falta la altura de product-info
como no está indicada solo esta usando la que necesita de acuerdo a su contenido
o usar alin-items: end; en .product para que los elemento de cada celda del grid vayan hasta el fondo de sus respectivas celdas

**Reply by Canti21** [reply]
Ok, lo intentare, gracias.

**Reply by carlos-gmx3** [reply]
Va, checa esta guía, te muestra las diferentes configuraciones que puedes usar para moldear tu grid
[A Complete Guide to Grid](https://css-tricks.com/snippets/css/complete-guide-grid/#prop-align-items)
